content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,239 @@
+│ └── exp-ad5b1 │  0.56191 │ 0.93345 │ 50         │ 2               │
+└───────────────┴──────────┴─────────┴────────────┴─────────────────┘
+```
+
+We are working on developing experiments to have features or documented patterns
+explicitly for grid search support, so definitely
+[share any feedback](https://github.com/iterative/dvc/issues/4283) to help drive
+the future direction of that!
+
+### [When importing/getting data from a repo, how do I provide credentials to the source repo remote storage without saving it into that Git repo?](https://discord.com/channels/485586884165107732/563406153334128681/830021022337073185)
+
+There's a bit of context behind this question that might give it more meaning.
+Here's the background information given by @EdAb in Discord:
+
+---
+
+I set up a private GitHub repo to be a data registry and I have set up a private
+Azure remote where I have pushed some datasets.
+
+I am now trying to read those datasets from another repository
+("my-project-repo"), using `dvc get` (e.g.
+`dvc get git@github.com:data-registry-repo.git path/data.csv`) but I get this
+error:
+
+```bash
+ERROR: failed to get 'path/data.csv' from 'git@github.com:data-registry-repo.git' - Authentication to Azure Blob Storage via default credentials (https://azuresdkdocs.blob.core.windows.net/$web/python/azure-identity/1.4.0/azure.identity.html#azure.identity.DefaultAzureCredential) failed.
+Learn more about configuration settings at <https://man.dvc.org/remote/modify>: unable to connect to account for Must provide either a connection_string or account_name with credentials!!
+```
+
+---
+
+Generally, there are two ways solve this issue:
+
+- [ENV vars](https://dvc.org/doc/command-reference/remote/modify)
+- Setup some options using the `--global` or `--system` flags to update the DVC
+  config
+
+If you're going to update the DVC config to include your cloud credentials, use
+the `dvc remote modify` command. Here's an example of how you can do that with
+Azure using the `--global` flag.
+
+```dvc
+$ dvc remote modify --global myremote connection_string 'mysecret'
+```
+
+You should initialize `myremote` in the config file with `dvc remote add` and
+remove the URL to rely on the one that comes from the repo being imported.
+
+This will modify the global config file, instead of the _.dvc/config_ file. You
+could also use the `--system` flag to modify the system file if that's necessary
+for your project. You can take a look at the specific
+[config file locations here](https://dvc.org/doc/command-reference/config).
+
+### [Is there any way to ensure that `dvc import` uses the cache from the config file and how can I keep the cache consistent for multiple team members?](https://discord.com/channels/485586884165107732/563406153334128681/827574712825413672)
+
+This is another great question where a little context might be useful.
+
+---
+
+I'm trying to import a dataset project into another DVC project, let's call it
```

Comment:
I read "let's call it" `dvcdata as if `another DVC project` has this name. But it's other way around `dvcdata` is being imported into some project (should we name it somehow here for simplicity) 

> May be it's me not reading english properly - feel free to correct me please

Reply:
I cleared that up so it reads better.🙂